operator/builtin/input/journald/journald.go

```diff
@@ -222,12 +222,35 @@ func (operator *JournaldInput) parseJournalEntry(line []byte) (*entry.Entry, str
 		return nil, "", errors.New("journald field for cursor is not a string")
 	}
 
-	entry, err := operator.NewEntry(body)
+	msg, ok := body["MESSAGE"]
+	if !ok {
+		return nil, "", errors.New("journald body missing MESSAGE field")
+	}
+	delete(body, "MESSAGE")
+
+	entry, err := operator.NewEntry(msg)
 	if err != nil {
 		return nil, "", fmt.Errorf("failed to create entry: %s", err)
 	}
-
 	entry.Timestamp = time.Unix(0, timestampInt*1000) // in microseconds
+
+	for k, v := range body {
+		switch k {
+		case "PRIORITY":
+			if err := addSeverity(entry, v); err != nil {
+				return nil, "", err
+			}
+		default:
+			str, ok := v.(string)
+			if !ok {
+				// attributes only supports strings at the moment
+				// https://github.com/open-telemetry/opentelemetry-log-collection/issues/190
+				continue
```

Comment:
Should we try to convert common types to a string? I think all values are strings in the first place, but I think we should avoid silently dropping a field in the event it is not a string.

Reply:
Yep, can do.  The options were to wait for #190 to be fixed (I've not looked at what is needed for this) at which point we'll be able to do this properly or convert everything to a string.